fv3core/utils/corners.py

```diff
@@ -70,6 +72,61 @@ def __call__(self, field: FloatField):
         self._copy_corners(self._corner_tmp, field)
 
 
+class CopyCornersXY:
+    """
+    Helper-class to copy corners corresponding to the Fortran functions
+    copy_corners_x and copy_corners_y
+    """
+
+    def __init__(
+        self,
+        grid_indexing: GridIndexing,
+        dims: Sequence[str],
+        y_field,
```

Comment:
I removed the type hint because I can't put Optional[FloatFIeld] because FloatField is not a real type. We talked about turning FloatField into a real type a while back, but haven't gotten around to it. I should update the docstring to convey more of the information that would be conveyed by this type hint.

Reply:
Updated.